menu.cpp

```diff
@@ -1180,8 +1180,8 @@ void HandleUI(void)
 			       menu, select, up, down,
 			       left, right, plus, minus);
 
-	switch (menustate)
-	{
+    switch (menustate)
+    {
```

Comment:
replacing tabs with spaces does not seem to be the right fix here since almost all of the code is indented with tabs

Reply:
you are right, sorry about that.